mkt/site/context_processors.py

```diff
@@ -14,7 +14,7 @@
 def i18n(request):
     return {'LANGUAGES': settings.LANGUAGES,
             'LANG': settings.LANGUAGE_URL_MAP.get(translation.get_language())
-                    or translation.get_language(),
+            or translation.get_language(),
```

Comment:
I think if you wrap the statement in parens it uses the formatting most people would want.

```python
    'LANG': (settings.LANGUAGE_URL_MAP.get(translation.get_language())
             or translation.get_language()),